dialogue-clients/src/main/java/com/palantir/dialogue/clients/ReloadingClientFactory.java

```diff
@@ -762,8 +596,5 @@ public String toString() {
     interface InternalDialogueChannelConfiguration {
```

Comment:
minor nit: do we need this type anymore?

Reply:
Good call, might as well remove it!